test-utils/generators/ModelGenerators.scala

```diff
@@ -17,35 +17,14 @@
 package generators
 
 import models.*
+import org.scalacheck.Arbitrary.arbitrary // scalafix:ok; need to keep this import cos it could be used by some scaffold
```

Comment:
For some reason this scalafix suppression comment disables the rest of this file. I've also tried with `scalafix:off` then `scalafix:on` and the same behaviour was observed.

However we do need to keep this since certain scaffold expect this import

Reply:
Discussed offline, we shouldn't enable both OrganizeImports & RemoveUnused, and since this appears to be a bug in current scalafix + scala versions we've agreed to leave it be for the time being